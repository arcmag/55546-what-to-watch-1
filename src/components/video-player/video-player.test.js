import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';

const mock = {
  poster: `poster.jpg`,
  src: `video.mp4`,
  video: `video.mp4`,
};

describe(`Test GenreList`, () => {
  it(`renderer`, () => {
    const tree = renderer.create(<VideoPlayer
      poster={mock.poster}
      src={mock.src}
      video={mock.video}
    />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

