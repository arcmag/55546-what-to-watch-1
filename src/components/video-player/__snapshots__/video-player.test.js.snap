// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test GenreList renderer 1`] = `
<video
  autoPlay={true}
  height="175"
  muted={true}
  poster="video.mp4"
  src="video.mp4"
  width="280"
/>
`;
